src/ui/components/NavAboutButton.js

```diff
@@ -1,16 +1,17 @@
 import { render } from '/modules.js'
-import { dispatch } from '/store/provider.js'
-import { types } from '/store/action-types.js'
+import { Icon } from './Icon.js'
 
-export const NavAboutButton = () => render`
-  <li id="info-link" class="controls-contaiter">
-    <button 
-      class="invisible-button item"  
-      ::click=${e => {
-        dispatch({ type: types.SHOW_INFO })
-      }}
+import { router } from '../../hashrouter.js'
+
+export const NavAboutButton = router(
```

Comment:
Try using something like `getParams` instead of `router` to get the matching route params.